polling/settings.py

```diff
@@ -20,7 +23,7 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '(3coz70ywrka%i2go(r$_ew^28@h*7g^c*g)f%n%nz%lxu77y9'
+SECRET_KEY =os.getenv('SECRET_KEY')
```

Comment:
Add instructions in the readme.md's how to setup section details of how to set this up